src/start.js

```diff
@@ -14,33 +14,37 @@
  *    - 'app:loaded' - Notify when client side app is loaded
  *    - 'download:completed' - update has finished downloading
  */
-const { app, ipcMain, dialog, BrowserWindow, session, Tray, nativeImage } = require('electron')
-const path = require('path')
-const url = require('url')
-const log = require('./lib/logger')
-const initMenu = require('./Menu')
-const config = require('./config.json')
-const { MINIMUM_AUTOSCAN_INTERVAL_SECONDS } = require('./constants')
-const settings = require('electron-settings')
-const serializeError = require('serialize-error')
-const initProtocols = require('./lib/protocolHandlers')
+import path from 'path'
+import { app, ipcMain, dialog, BrowserWindow, session, Tray, nativeImage } from 'electron'
+import url from 'url'
+import log from './lib/logger'
+import initMenu from './Menu'
+import config from './config.json'
+import { MINIMUM_AUTOSCAN_INTERVAL_SECONDS } from './constants'
+import settings from 'electron-settings'
+import serializeError from 'serialize-error'
+import initProtocols from './lib/protocolHandlers'
+import loadReactDevTools from './lib/loadReactDevTools'
+import iconFinder from './lib/findIcon'
+import * as scripts from './lib/scripts'
+import startGraphQLServer from './server'
+import { IS_MAC, IS_WIN } from './lib/platform'
+import AutoLauncher from './AutoLauncher'
+import updateInit from './updater'
+
 const env = process.env.STETHOSCOPE_ENV || 'production'
-const loadReactDevTools = require('./lib/loadReactDevTools')
-const findIcon = require('./lib/findIcon')(env)
-const startGraphQLServer = require('../server')
+const findIcon = iconFinder(env)
 const IS_DEV = env === 'development'
-const { IS_MAC, IS_WIN } = require('./lib/platform')
-const AutoLauncher = require('./AutoLauncher')
-
 const disableAutomaticScanning = settings.get('disableAutomaticScanning')
 
+scripts.init()
```

Comment:
This shouldn't be needed since we are running the scripts as needed, correct?

Reply:
This just preloads the scripts so that they can be executed, it does not execute any scripts.

Reply:
Does precompiling still make sense as we are compiling and running on each kmd call?